src/pages/articles/_layouts/ArticleLayout.astro

```diff
@@ -159,6 +192,44 @@ const structuredData = JSON.stringify({
         <div class="hidden lg:block"><SocialShare /></div>
         <ArticleSeries {article} open={true} />
         <AuthorWide {author} />
+
+        {
+          relatedArticles.length > 0 && (
+            <RelatedArticles length={relatedArticles.length} client:load>
+              {relatedArticles.length >= 1 && (
+                <ArticlePreview
+                  slot="firstArticle"
```

Comment:
This seems pretty tedious. I guess you have to do this because the slot names are static?

Reply:
Will replace with map

Reply:
Map will not work with static slot names here.